tests/ConfigTest.php

```diff
@@ -223,6 +223,72 @@ public function testConfigWithSshTunnel(): void
         $config = new Config($configurationArray, new ActionConfigRowDefinition());
         $this->assertEquals($configurationArray, $config->getData());
     }
+
+    public function testConfigWithSshTunnelEnabled(): void
+    {
+        $configurationArray = [
+            'parameters' => [
+                'data_dir' => '/code/tests/Keboola/DbExtractor/../../data',
+                'extractor_class' => 'MySQL',
+                'db' => [
+                    'host' => 'mysql',
+                    'user' => 'root',
+                    '#password' => 'rootpassword',
+                    'database' => 'test',
+                    'port' => 3306,
+                    'initQueries' => [],
+                    'ssh' => [
+                        'enabled' => true,
+                        'user' => 'root',
+                        'sshHost' => 'sshproxy',
+                        'sshPort' => '22',
+                        'localPort' => '33306',
+                        'keys' => ['public' => 'anyKey', '#private' => 'anyKey'],
+                        'maxRetries' => 10,
+                        'compression' => false,
+                    ],
+                ],
+            ],
+        ];
+
+        $config = new Config($configurationArray, new ActionConfigRowDefinition());
+        $this->assertEquals($configurationArray, $config->getData());
+    }
+
+    public function testConfigWithSshTunnelEnabledMissingPrivateKey(): void
```

Comment:
ještě prosím test kdy je enabled na false a chybí nějaký povinný klíče a projde to

Reply:
Jeden tam je, kde úplně chybí `enabled`, přidal jsem ještě jeden, kde `enabled` = `false`.